nixos/modules/services/networking/corerad.nix

```diff
@@ -0,0 +1,46 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.services.corerad;
+in {
+  meta = {
+    maintainers = with maintainers; [ mdlayher ];
+  };
+
+  options.services.corerad = {
+    enable = mkEnableOption "CoreRAD IPv6 NDP RA daemon";
+
+    configFile = mkOption {
+      type = types.path;
+      example = "${pkgs.corerad}/etc/corerad/corerad.toml";
+      description = "Path to CoreRAD TOML configuration file.";
+    };
+
+    package = mkOption {
+      default = pkgs.corerad;
+      defaultText = "pkgs.corerad";
```

Comment:
And ideally this should also be `defaultText = literalExample "pkgs.corerad"`. This renders better in the manual (`nix-build nixos/release.nix -A manualHTML.x86_64-linux`)

Reply:
Updated.